language/vm/vm-runtime/src/execution_context.rs

```diff
@@ -94,6 +90,8 @@ pub trait InterpreterContext {
     fn deduct_gas(&mut self, amount: GasUnits<GasCarrier>) -> VMResult<()>;
 
     fn remaining_gas(&self) -> GasUnits<GasCarrier>;
+
+    fn exists_module(&self, m: &ModuleId) -> bool;
```

Comment:
it's convenient to add this here for now. The "conversation" between the Libra and VM layer is evolving and this may go elsewhere....